glam/api/management/commands/import_desktop_sample_counts.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+import os
+import tempfile
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from google.cloud import storage
+
+from glam.api import constants
+from glam.api.models import FirefoxSampleCounts
+
+
+# For logging
+FILENAME = os.path.basename(__file__).split(".")[0]
+
+
+def log(message):
+    print(f"{datetime.datetime.now().strftime('%x %X')} - {FILENAME} - {message}")
+
+
+class Command(BaseCommand):
+
+    help = "Imports sample counts"
```

Comment:
This file, and `import_glean_sample_counts`, are almost identical to `import_user_counts.py` and `import_glean_counts.py`. Would it make more sense to consolidate adding sample and user counts into just one command, e.g `import_desktop_counts` for Desktop and `import_glean_counts` for Glean?

Reply:
I did think about this. My rationale behind separating them was we could run them independently (eliminating  dependency on each other)